src/span.rs

```diff
@@ -1,3 +1,7 @@
+use crate::{Arena, Handle, UniqueArena};
+use std::borrow::Cow;
+use std::error::Error;
+use std::fmt::{Display, Formatter};
```

Comment:
let's just include `fmt` and refer to members as `fmt::Xxx` - that's what other modules do in this repo